packages/apollo-voyager-server/src/audit/DefaultAuditLogger.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLResolveInfo } from 'graphql'
+import { AuditLogger } from './AuditLogger'
+
+export class DefaultAuditLogger implements AuditLogger {
+
+  public logResolverCompletion (msg: string, success: boolean, obj: any, args: any, context: any, info: GraphQLResolveInfo): void {
```

Comment:
It actually begs to have interface and have ? on the parameter that is different. Then we can reuse. No strong opinion on it, but it will be much cleaner.

Reply:
As I mentioned before, there is a very important separation between these two functions. The user should not be able to log a resolver completions. They should not be able to pass the `success` thing at all.

I see these as two functions that serve two totally different purposes.

Reply:
@darahayes what's the effect on Kibana dashboards now?

Confirm what I understood: "success" is the field we should look for existence for the resolver execution graphs such as "number of queries executed".

Reply:
not the best place to ask this question :)